spec/api-browser-window-spec.js

```diff
@@ -1106,6 +1106,90 @@ describe('BrowserWindow module', function () {
       })
       w.loadURL('file://' + path.join(fixtures, 'api', 'close-beforeunload-empty-string.html'))
     })
+
+    it('fires for each close attempt', function (done) {
+      var closeCount = 5
+
+      var timeoutHandle = setTimeout(() => {
```

Comment:
The `timeoutHandle` can lead to a flaky spec, better to remove it. Sometimes the spec may take longer than expected. We do have an overall timeout.